src/main/scala/wdl4s/cwl/model/CWLType.scala

```diff
@@ -0,0 +1,55 @@
+package broad.cwl.model
+
+import enumeratum._
+import mouse.boolean._
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.string._
+import eu.timepit.refined._
+import eu.timepit.refined.auto._
+
+sealed abstract class CWLType(override val entryName: String) extends EnumEntry
```

Comment:
Oh, yeah. I am a fan of sticking with camel case (or whatever) in the face of acronyms. We do a mix though